app/assets/javascripts/bulk_updates.coffee

```diff
@@ -1,13 +1,39 @@
+displayHelpText = (searchFieldElement) ->
+  # The select element that lists the fields to search on
+  $selectedFieldData = searchFieldElement.find('option:selected').data()
+
+  # The query (value) field and its form-description
+  $queryField = searchFieldElement.closest('div').find('#bulk-updates-search-field')
+  $queryFieldDescription = $('#bulk-update-query-description')
+
+  # Set the form-description if the field has one
+  if($selectedFieldData.hasOwnProperty('description'))
+    $queryFieldDescription.text($selectedFieldData['description'])
+  else
+    $queryFieldDescription.text('')
+
+  searchFieldElement.next('.form-description').remove()
+
+  if($selectedFieldData.hasOwnProperty('supports_wildcard') && $selectedFieldData['supports_wildcard'] == true)
+    $('<p>').addClass('form-description')
+      .text('This field supports wildcard searching. Add `*` to your search to return more results.')
```

Comment:
Based on a quick search and read of other applications' same feature the following seems to align with the feature. 

'This field supports wildcard searches. Use an asterisk (*) to find collections that match zero or more characters at the beginning, middle or end of your term.'